### ISSUE TITLE
Microsoft Edge extension?

### ISSUE THREAD

**Issue by BenHocking** [golopot/tex-to-unicode#32]
Any chance for a Microsoft Edge extension?

(Yeah, I understand the irony of wanting to do TeX on MS Edge.)

**Comment by xworld21** [comment]
I understand porting the extension from Chrome to Edge should be fairly easy (but maybe not!), I am happy to give it a try if @golopot is interested.

**Comment by golopot** [comment]
Yes I guess the porting should be easy. I can do it when I get some spare time.

**Comment by yehoshuapw** [comment]
now, since Edge is chromium - this should simply work without any changes.
